husc/screens/cellomics.py

```diff
@@ -0,0 +1,366 @@
+"""Feature computations and other functions for Cellomics screens.
+"""
+
+import os
+import collections as coll
+from skimage import io
+import numpy as np
+from cytoolz import groupby
+
+
+def listdir_fullpath(path):
+    """Extended os.listdir that lists the full directory instead of just the
+    filename.
+
+    Parameters
+    ----------
+    dir : string
+        The query directory.
+
+    Returns
+    -------
+    path : list of string
+        A list of files in the query directory with their full path.
+    """
+    path = [os.path.join(path, fn) for fn in os.listdir(path)]
+    return path
+
+
+def batch_stitch_stack(file_dict, output, order=[2, 1, 0], bit=None):
+    """Run snail stitch and concatenate the channels across a set of images.
+
+    This function takes the (plate, well) dictionary built using the
+    ``make_key2file`` function. Images are grouped according to their channel,
+    stitched together and stacked into a single 3-channel image. Images
+    are re-scaled and saved to a user specified output directory. Images
+    are saved to directories according to their plate number.
+
+    Parameters
+    ----------
+    file_dict : dict { tuple (plate, well) : list of strings }
+        The dictionary mapping the (plate, well) tuple to a list of image
+        files. This dictionary is built using the ``make_key2file`` function.
+    output : string
+        The directory to output the stitched and concatenated images to.
+    order : list of int, optional
+        The order the channels should be in in the final image. Default
+        [0, 1, 2].
+    bit : string, optional
```

Comment:
`bit_depth` is more descriptive. In fact, I might say, `target_bit_depth`.

btw, `string`? How about `int in {8, 16}, optional`.

Finally, the description is a bit messy, how about, "If `None` (default), perform no rescaling. Otherwise, rescale to occupy the dynamic range of the target bit depth."